docs/contributing.md

```diff
@@ -1,52 +1,81 @@
 # 🤝 Contributing
 
-> [!WARNING]
-> This is out of date at present and will be updated soon
+Thank YOU for even considering contributing 💖
 
-## Add an event
+## Adding content (events/resources/etc)
 
-Add a file inside the [`_events`](_events) directory. Copy a previous file as a template, and make sure to include the right time zone for The Netherlands!
+Content can be added in two different ways:
 
-## Add a blog post (inside /blog)
+1. Via the admin site: https://techwerkers.nl/admin
+1. Running `decap-server` locally (see heading below) and commiting the changes instead.
 
-Add a file inside the [`_blog`](_blog) directory. Copy a previous file as a template. If an author does not exist, add one inside [`_data/authors.yml`](_data/authors.yml). A name is the only thing necessary, but photo is optional too.
+> [!INFO]
+> The admin site will only be accessible to those that have access to the netlify admin. You can ask for access in our
+> slack channel.
 
-## Add a press mention
+### Content Guidelines
 
-Inside [`_data/press.yml`](_data/press.yml) file, add a media entry, with date format in `YYYY-MM-DD`.
+General writing guidelines:
 
-## Translation
+- Use as little jargon as possible and/or make sure to define it where necessary. We want as many people to be able to
+  understand the content as possible.
```

Comment:
Completely on board with the point! Though maybe could also consider swapping it to the positive ('use plain language') which maybe sounds more constructive? Maybe smth like:

```suggestion
- Use plain language wherever you can. We want as many people to be able to
  understand the content as possible.
```

Reply:
oooo I like how you phrased this. This is one of those linguistic hacks I am going to borrow. Instead of "less X", "more Y"  :sweat_smile: